src/models/UserModel.js

```diff
@@ -1,15 +1,27 @@
-// api/v1/models/userModel.js
-import { pool } from "../config/database.js";
+import { dbConnection } from "../config/database.js";
+import logger from "../utils/logger.js";
 
 export const createUser = async (firstname, lastname, email, password) => {
 	const query =
 		"INSERT INTO users (firstname, lastname, email, password) VALUES ($1, $2, $3, $4) RETURNING *";
 	const values = [firstname, lastname, email, password];
 
 	try {
-		const result = await pool.query(query, values);
+		const result = await dbConnection.query(query, values);
```

Comment:
A function was already created `dbConnection` for interacting with the DB. You don't have to introduce a new methodology if one is already established in a project.